frontend/src/types/user.ts

```diff
@@ -0,0 +1,29 @@
+// export class User {
+//   id: number;
+//   username: string;
+//   email: string;
+//   first_name: string;
+//   last_name: string;
+//
+//   constructor(data: { id: number, username: string, email: string, first_name: string, last_name: string }) {
+//     this.id = data.id;
+//     this.username = data.username;
+//     this.email = data.email;
+//     this.first_name = data.first_name;
+//     this.last_name = data.last_name;
+//   }
+//
+//   get display_name(): string {
+//     if (this.first_name && this.last_name) return `${ this.first_name } ${ this.last_name }`;
+//     else return this.username;
+//   }
+// }
```

Comment:
You know the drill: is this still useful?

Reply:
Thanks for the remind ^^
I removed the interface to use this class. Globally, I'd like to replace a lot of interfaces that reprensent API data by object to simplify its use and common data transformation used around the app.